### ISSUE TITLE
[BUG] Type error on rename columns

### ISSUE THREAD

**Issue by lmeyerov** [rapidsai/cudf#3285]
**Describe the bug**

On a call `src_edges.rename(columns={'nid': 'src'}, copy=False)`, we get error:

```
File "/opt/conda/envs/rapids/lib/python3.7/site-packages/cudf/core/dataframe.py", line 1690, in rename
out_column = mapper[key] + ("cudf_" + str(postfix),)
TypeError: can only concatenate str (not "tuple") to str
```

**Steps/Code to reproduce bug**

I wasn't able to repro on an isolated test case... but I'm not sure how that line ever works: `'x' + ('x',)` is a type error.

**Environment overview (please complete the following information)**

cudf 0.10 (docker), and same line of code is in cudf head (https://github.com/rapidsai/cudf/blob/branch-0.11/python/cudf/cudf/core/dataframe.py#L1768)

**Comment by lmeyerov** [comment]
(Worked around via `.assign(**{'to': from})`, but a regression from working 0.7 cudf code)

**Comment by galipremsagar** [comment]
Fixed by #4633
